artiq/coredevice/sampler.py

```diff
@@ -129,6 +129,32 @@ def sample_mu(self, data):
             data[i] = val >> 16
             val &= 0xffff
             data[i - 1] = -(val & mask) + (val & ~mask)
+            
+    
```

Comment:
There are a bunch of checkboxes that you haven't completed yet. One of them is style. Please make some effort to propose quality code that matches the prevalent style and orthography.